packages/bits/src/lib/select-v2/option-key-control.service.ts

```diff
@@ -45,6 +45,15 @@ export class OptionKeyControlService<T extends IOption> {
         this.keyboardEventsManager.skipPredicate(predicate);
     }
 
+    public scrollToOption(options: ScrollIntoViewOptions): void {
```

Comment:
It's just a suggestion, but I think `scrollToActiveItem` might be a better name for this method, and since it's newly public, it wouldn't be a breaking change to rename it.

Reply:
done